layers/meta-balena-fsl-arm/conf/samples/local.conf.sample

```diff
@@ -2,13 +2,16 @@
 #MACHINE ?= "nitrogen6x"
 #MACHINE ?= "solidrun-imx6"
 #MACHINE ?= "nitrogen6xq2g"
+#MACHINE ?= "nitrogen8mm"
+#MACHINE ?= "nitrogen8mm-dwe"
 
 # More info meta-resin/README.md
 #TARGET_REPOSITORY ?= ""
 #TARGET_TAG ?= ""
 
 BALENA_STORAGE_nitrogen6xq2g = "overlay2"
 BALENA_STORAGE_nitrogen8mm = "overlay2"
+BALENA_STORAGE_nitrogen8mm-dwe = "overlay2"
```

Comment:
The git commit log is misleading a little as it implies we switched to overlay2 for other machines that were using something else before this change

Reply:
Re-worded, thanks